api/oss/src/apis/fastapi/testsets/router.py

```diff
@@ -59,6 +59,26 @@
     SimpleTestsetsService,
 )
 
+
+def _infer_columns_from_revision_data(revision_data: Any) -> List[str]:
```

Comment:
@ardaerzin 
I am not sure about this change.
We said we'd do this for legacy `testsets`, no? This is preview `testsets`, right?